### PR TITLE
Documentation fixes including updated Contribution pages

### PR DESCRIPTION
**Pull request by ooeygui**

### CODE REVIEW

**Comment by anthony-ngu** [comment]
Why are some of the commits from yesterday being recommitted?

These ones:
Missing image was due to case sensitivity
Documentation Bug Fixes

Also, why is the README being made into a markdown file? and if this is what you want to do, each step should be separate lines.

**Comment by ooeygui** [comment]
README should always have been a markdown file

Because I had an existing fork, I rebased. This means my code was brought into line with the root. You are just seeing that loop.

File: Contribute.md [hunk 1]
```diff
@@ -21,75 +21,73 @@ We're actively accepting work for the following areas:
...
-###Making a change
-1. Open GitBash
-1. Create a branch for your feature
-
-   {% highlight bash %}
-   git pull upstream master:<new branch name>`
-   {% endhighlight %}
-  
-   For Example:
+1. Now that you have an account, you need to install the [GitHub app](https://windows.github.com/) on your computer.
```
Comment (JustAddWires) [inline Contribute.md:26]: The first item on the first page of the GitHub app documentation was the link to install...

File: Contribute.md [hunk 2]
```diff
@@ -21,75 +21,73 @@ We're actively accepting work for the following areas:
...
-1. Make your edits, build and test.
-1. Add your edits and commit
+___
 
-   {% highlight bash %}
-git add .
-git commit -v -a -m "<Describe the changes made in this commit>"
-   {% endhighlight %}
+##Making changes
+If you are editing a fork of ms-iot/content, please submit pull request off of gh_pages.
```
Comment (JustAddWires) [inline Contribute.md:36]: Should mention how to submit pull requests for ms-iot/galileo-sdk

**Pull request merged**